### ISSUE TITLE
Calls filtering (ie do not log calls from...)

### ISSUE THREAD

**Issue by papaja** [blak3r/yaai#32]
How would one go about filtering what calls are being recorded? For instance I do not want fax sending/receiving events to be recorded or some other extensions to be monitored.

**Comment by blak3r** [comment]
I'm not planning on implementing any features like this.  It'll have to
come from the community.

If the calls don't match a person's extension they just get logged to the
"Administrator" account.  You could create a search that searched by phone
number and delete unwanted calls periodically.

On Wed, Jul 4, 2012 at 9:08 AM, papaja <
reply@reply.github.com

**Comment by papaja** [comment]
I've modified line: if ($e['Event'] == 'Dial' && $e['SubEvent'] != 'End') to be: if ($e['Event'] == 'Dial' && $e['SubEvent'] != 'End' && strpos($e['Channel'],'808')===FALSE ) and it excludes what I wanted.

Thanks for reply.

**Comment by papaja** [comment]
I have just noticed that you've added (ie do not log calls from...) which sounds like ignore calls from some external caller. I was more thinking along the lines (ie do not log calls to or from internal local...)

**Comment by blak3r** [comment]
@papaja your fix is interesting.  I forget exactly how the asteriskLogger works... but I believe a call entry is still being created.  In the Dial End event, I believe we just update the call record that was created previously in the Dial/Begin event.    Maybe we delete it there though too I forget. 

RE: my change to your title.  The plugin shouldn't be logging internal to internal calls as it is.  If it is you probably need to adjust a regex.

**Comment by papaja** [comment]
Blake,
I'm willing to send you my asteriskLogger.php so you can see and adopt maybe some pieces. It is also created as "works for me" so I don't dare uploading it as alternative.

**Comment by blak3r** [comment]
Yeah please share. I'd be great if you could isolate your changes and create a pull request.  The amount of time it takes for me to do it is greater then the time it'd be for you.  But yes I'd definitely rather have a copy then not. 

~blake

**Comment by papaja** [comment]
How can I send you a copy?

On Sat, Jul 14, 2012 at 2:29 PM, Blake Robertson <
reply@reply.github.com

> wrote:
> 
> Yeah please share. I'd be great if you could isolate your changes and
> create a pull request.  The amount of time it takes for me to do it is
> greater then the time it'd be for you.  But yes I'd definitely rather have
> a copy then not.
> 
> ~blake
>

**Comment by blak3r** [comment]
You can email it. My address is on my github profile. 

~blake
